pdfminer/pdffont.py

```diff
@@ -76,7 +78,33 @@ def get_widths2(seq):
 #assert get_widths2([1,2,3,4,5]) == {1:(3, (4,5)), 2:(3, (4,5))}
 #assert get_widths2([1,[2,3,4,5],6,[7,8,9]]) == {1:(2, (3,4)), 6:(7, (8,9))}
 
-
+def computeStat(it):
```

Comment:
Function names should be lowercase with underscores for extra readability: https://www.python.org/dev/peps/pep-0008/#function-and-variable-names